.circleci/config.yml

```diff
@@ -110,7 +110,7 @@ jobs:
           NODE_ENV: test
           AWS_XRAY_LOG_LEVEL: silent
           AWS_XRAY_CONTEXT_MISSING: LOG_ERROR
-      - image: pocket/snowplow-micro:prod
+      - image: pocket/snowplow-micro:latest
```

Comment:
`latest` points to dev. we should be running circleCI test against prod, so we can be confident that our snowplow events are emitted against prod schema